cmd/dao_tests/pubkey.go

```diff
@@ -0,0 +1,208 @@
+package main
+
+import (
+	"context"
+	"testing"
+
+	"github.com/RHEnVision/provisioning-backend/internal/dao"
+	_ "github.com/RHEnVision/provisioning-backend/internal/dao/sqlx"
+	"github.com/RHEnVision/provisioning-backend/internal/models"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/RHEnVision/provisioning-backend/internal/config"
+	"github.com/RHEnVision/provisioning-backend/internal/db"
+)
+
+func createPk() []*models.Pubkey {
+	return []*models.Pubkey{{
+		ID:        10,
+		AccountID: 2,
+		Name:      "testkey",
+		Body:      "sha-rsa body",
+	}}
+}
+
+func updatePk() []*models.Pubkey {
+	return []*models.Pubkey{{
+		ID:        1,
+		AccountID: 1,
+		Name:      "updated-key",
+		Body:      "updated body",
+	}}
+}
+
+func TearDown(t *testing.T) {
+	config.Initialize()
+
+	err := db.Initialize()
+	if err != nil {
+		t.Errorf("Error initializing database.")
+		return
+	}
+
+	err = db.Seed("drop_all")
+	if err != nil {
+		t.Errorf("Error purging the database")
+		return
+	}
+
+	err = db.Migrate()
+	if err != nil {
+		t.Errorf("Error running migration")
+		return
+	}
+}
+
+func CreatePubkeyTest(t *testing.T) {
```

Comment:
Go convention is to name Test functions with Test prefix, not suffix, is there a reason we are using suffix instead here?